app/pkg/env/env.go

```diff
@@ -42,12 +42,13 @@ type config struct {
 		WriteTimeout time.Duration `env:"HTTP_WRITE_TIMEOUT,default=10s,strict"`
 		IdleTimeout  time.Duration `env:"HTTP_IDLE_TIMEOUT,default=120s,strict"`
 	}
-	Port       string `env:"PORT,default=3000"`
-	HostMode   string `env:"HOST_MODE,default=single"`
-	HostDomain string `env:"HOST_DOMAIN"`
-	BaseURL    string `env:"BASE_URL"`
-	Locale     string `env:"LOCALE,default=en"`
-	JWTSecret  string `env:"JWT_SECRET,required"`
+	Port                        string `env:"PORT,default=3000"`
+	HostMode                    string `env:"HOST_MODE,default=single"`
+	HostDomain                  string `env:"HOST_DOMAIN"`
+	BaseURL                     string `env:"BASE_URL"`
+	Locale                      string `env:"LOCALE,default=en"`
+	JWTSecret                   string `env:"JWT_SECRET,required"`
+	PostCreationWithTagsEnabled bool   `env:"POST_CREATION_WITH_TAGS_ENABLED,default=false"`
```

Comment:
Yeah this looks good. I was contemplating if we should have a nested `AppFeatures` struct, and move this into there so that we could add other things in future, but I think we _probably_ don't need to do that...